packages/gallery/src/components/item/itemView.js

```diff
@@ -198,27 +198,35 @@ class ItemView extends GalleryComponent {
       e.preventDefault();
     }
 
-    if (this.shouldShowHoverOnMobile()) {
+    if (
+      this.shouldShowHoverOnMobile() &&
+      this.props.styleParams.hoveringBehaviour !==
```

Comment:
maybe use destructure here: const { hoveringBehaviour } = this.props.styleParams